bin/sozo/src/commands/options/transaction.rs

```diff
@@ -13,11 +13,17 @@ pub struct TransactionOptions {
     pub fee_estimate_multiplier: Option<f64>,
 
     #[arg(short, long)]
-    #[arg(help = "Wait until the transaction is accepted by the sequencer, returning the receipt.")]
+    #[arg(help = "Wait until the transaction is accepted by the sequencer, returning the status \
+                  and hash.")]
     #[arg(long_help = "Wait until the transaction is accepted by the sequencer, returning the \
-                       receipt. This will poll the transaction status until it gets accepted or \
-                       rejected by the sequencer.")]
+                       status and the hash. This will poll the transaction status until it gets \
+                       accepted or rejected by the sequencer.")]
     pub wait: bool,
+
+    #[arg(short, long)]
+    #[arg(help = "If --wait is set, returns the full transaction receipt.")]
+    #[arg(long_help = "If the --wait flag is set, returns the full transaction receipt.")]
```

Comment:
```suggestion
    #[arg(long_help = "If --wait is set, returns the full transaction receipt. Otherwise, it is a no-op.")]
```

lets just reuse the short help